runtime/common/config/pallets/governance.rs

```diff
@@ -0,0 +1,281 @@
+	}
+
+	#[cfg(feature = "runtime-benchmarks")]
+	fn try_successful_origin() -> Result<O, ()> {
+		OneThirdsFellowship::try_successful_origin()
+	}
+}
+
+pub type TechnicalCollective = pallet_collective::Instance3;
+impl pallet_collective::Config<TechnicalCollective> for Runtime {
+	type RuntimeOrigin = RuntimeOrigin;
+	type Proposal = RuntimeCall;
+	type RuntimeEvent = RuntimeEvent;
+	type MotionDuration = TechnicalMotionDuration;
+	type MaxProposals = TechnicalMaxProposals;
+	type MaxMembers = TechnicalMaxMembers;
+	type DefaultVote = pallet_collective::PrimeDefaultVote;
+	type WeightInfo = pallet_collective::weights::SubstrateWeight<Runtime>;
+}
+
+pub type TechnicalCommitteeMember = pallet_collective::EnsureMember<AccountId, TechnicalCollective>;
+
+pub type RootOrTechnicalCommitteeMember =
+	EitherOfDiverse<EnsureRoot<AccountId>, TechnicalCommitteeMember>;
+
+pub type AllTechnicalCommittee =
+	pallet_collective::EnsureProportionAtLeast<AccountId, TechnicalCollective, 1, 1>;
+
+pub type RootOrAllTechnicalCommittee =
+	EitherOfDiverse<EnsureRoot<AccountId>, AllTechnicalCommittee>;
+
+pub type CouncilCollectiveMembership = pallet_membership::Instance1;
+impl pallet_membership::Config<CouncilCollectiveMembership> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type AddOrigin = RootOrAllCouncil;
+	type RemoveOrigin = RootOrAllCouncil;
+	type SwapOrigin = RootOrAllCouncil;
+	type ResetOrigin = EnsureRoot<AccountId>;
+	type PrimeOrigin = EnsureRoot<AccountId>;
+	type MembershipInitialized = Council;
+	type MembershipChanged = Council;
+	type MaxMembers = CouncilMaxMembers;
+	type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
+}
+
+pub type FellowshipCollectiveMembership = pallet_membership::Instance2;
+impl pallet_membership::Config<FellowshipCollectiveMembership> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type AddOrigin = RootOrTechnicalCommitteeMember;
+	type RemoveOrigin = RootOrTechnicalCommitteeMember;
+	type SwapOrigin = RootOrTechnicalCommitteeMember;
+	type ResetOrigin = EnsureRoot<AccountId>;
+	type PrimeOrigin = EnsureRoot<AccountId>;
+	type MembershipInitialized = Fellowship;
+	type MembershipChanged = Fellowship;
+	type MaxMembers = FellowshipMaxMembers;
+	type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
+}
+
+pub type TechnicalCollectiveMembership = pallet_membership::Instance3;
+impl pallet_membership::Config<TechnicalCollectiveMembership> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type AddOrigin = RootOrHalfCouncil;
+	type RemoveOrigin = RootOrHalfCouncil;
+	type SwapOrigin = RootOrHalfCouncil;
+	type ResetOrigin = EnsureRoot<AccountId>;
+	type PrimeOrigin = EnsureRoot<AccountId>;
+	type MembershipInitialized = TechnicalCommittee;
+	type MembershipChanged = TechnicalCommittee;
+	type MaxMembers = TechnicalMaxMembers;
+	type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
+}
+
+impl pallet_scheduler::Config for Runtime {
```

Comment:
As I remember, one of the discussion's key points was making democracy execute calls on behalf of some other origin, but here I see it still works with a standard scheduler, which runs calls as root. Do I miss something?

Reply:
We decided to use the plain scheduler for democracy for safety reasons.

Reply:
But how do we prevent a malicious transaction from being executed?

Reply:
This scheduler is meant to be used **solely** by the democracy.

Reply:
@kozyrevdev

Reply:
Summary of a private conversation with Yaroslav: the Fellowship solves this problem.